aci/resource_aci_cloudctxprofile.go

```diff
@@ -98,13 +99,30 @@ func resourceAciCloudContextProfile() *schema.Resource {
 				Optional:    true,
 				Description: "hub network to enable transit gateway",
 			},
+			"cloud_brownfield": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Import Brownfield Virtual Network",
+				ForceNew:    true,
```

Comment:
Can you please confirm if ForceNew: true is required here?
If it is not we may have to modify the Update function as well.
I'll need to check the autogen script again

Reply:
yes it's required